style.css

```diff
@@ -329,6 +330,18 @@ button:active {
   gap: 6px;
 }
 
+.works_card_history ul {
+  display: flex;
+  gap: 12px;
+}
```

Comment:
- [ ] Good job adjusting the layout of the mini description in the popup but there is still some room for adjustment as I noticed you're forgetting to add the proper color as well as the bullet points between each text, so please make sure to update this in order to have a good replication of the original design.
- Figma design:
  ![image](https://github.com/katarighe/my-portfolio/assets/104968236/246e6588-9f43-41b1-98d0-fd4752f341e1)
- Your design:
  ![image](https://github.com/katarighe/my-portfolio/assets/104968236/5ef6962f-fb60-4670-8a94-904df7855fbc)